nimbus-core/src/main/java/com/antheminc/oss/nimbus/domain/model/config/builder/internal/DefaultEntityConfigBuilder.java

```diff
@@ -116,6 +120,22 @@ public DefaultEntityConfigBuilder(BeanResolverStrategy beanResolver, Map<String,
 			throw new InvalidConfigException("Persistable Entity: "+mConfig.getReferredClass()+" must be configured with @Id param which has Repo: "+mConfig.getRepo());
 		}
 		
+		if(domainSet == null)
+			return mConfig;
+		
+		if(CollectionUtils.isEmpty(domainSet.get(Scope.remote.name())) && CollectionUtils.isEmpty(domainSet.get(Scope.local.name())))
```

Comment:
Dont see any implementation for Scope.local. Is this check redundant given the check right after this line (#130)?